mantidproject/mantid
# Dev docs: MVP- and GUI-related pages

As part of #22082, the documentation related to MVP pattern and GUI design in Python was moved to the new `dev-docs` directory. The example calculator GUI, which was previously shown as a code listing in the documentation, was added to the `scripts` directory so that it can be played with more easily.

**To test:**

Decide whether `scripts` is the best place for `MVPExample` - if it's not, suggest where might be better. Build the docs, make sure everything looks nice.

Refs #22082 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
